.travis.yml

```diff
@@ -0,0 +1,66 @@
+########################################################################
+## Travis CI config for LimeSuite
+##
+## * not all components will be configured
+## * the matrix builds Release and Debug
+## * run the self tests after install
+########################################################################
+
+sudo: required
+dist: trusty
+
+language: cpp
+
+compiler:
+  - clang
+  - gcc
+
+os:
+  - linux
+  - osx
+
+env:
+  global:
+    - INSTALL_PREFIX=/usr/local
+    - SOAPY_SDR_BRANCH=master
+  matrix:
+    - BUILD_TYPE=Debug
+    - BUILD_TYPE=Release
+
+# whitelist
+branches:
+  only:
+    - master
+    - stable
+
+before_install:
+  
+  
+  # update after package changes
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get -qq update; fi;
```

Comment:
Since I think travis is on ubuntu trusty, the default cmake is too old. We need cmake 3 series: `sudo add-apt-repository ppa:george-edison55/cmake-3.x --yes` https://github.com/travis-ci/travis-ci/issues/4631